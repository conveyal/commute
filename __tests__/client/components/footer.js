/* global describe, expect, it */

import {mount} from 'enzyme'
import {mountToJson} from 'enzyme-to-json'
import React from 'react'

import Footer from '../../../client/components/footer'

describe('Container > Footer', () => {
  it('renders correctly', () => {
    // mount component
    const tree = mount(
      <Footer />
    )
    expect(mountToJson(tree)).toMatchSnapshot()
  })
})
