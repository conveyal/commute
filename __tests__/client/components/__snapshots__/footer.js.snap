// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container > Footer renders correctly 1`] = `
<Footer>
  <footer>
    <Grid
      bsClass="container"
      componentClass="div"
      fluid={false}
    >
      <div
        className="container"
      >
        <p
          className="text-center text-muted"
        >
          <span
            aria-label="Copyright"
            role="img"
          >
            Â©
          </span>
          <a
            href="http://conveyal.com"
          >
            Conveyal
          </a>
        </p>
      </div>
    </Grid>
  </footer>
</Footer>
`;
